src/app/pages/HomePage/HomePage.tsx

```diff
@@ -55,6 +56,21 @@ const HomePage = ({ pageData }: HomePageProps) => {
   return (
     <>
       <ChartbeatAnalytics title={title} />
+      <Helmet>
+        <script
+          src="https://static.bbci.co.uk/frameworks/requirejs/0.13.0/sharedmodules/require.js"
+          type="text/javascript"
+        />
+        <script type="text/javascript">
+          {`
+      bbcRequireMap = {
+        "bump-4": "https://emp.bbci.co.uk/emp/bump-4/bump-4"
+      };
+      require({ paths: bbcRequireMap, waitSeconds: 30 });
+    `}
+        </script>
+      </Helmet>
```

Comment:
The `MediaLoader` was bringing in `require.js` conditionally for the BUMP/SMP player on the Home Page when the play button within the `modal` was clicked. That caused a sync issue with the module in the browser (failed to be downloaded in the network tab). As a temporary fix, I couldn't find any other examples to reference against so have placed react `Helmet` at the top level of the `home` page so that `require.js` is available for `smp` to run the videos. As a result, the videos then load properly. We should consider a more production-friendly solution that might mean adapting `MediaLoader` so that in use cases like this where video playback is hidden within a component, the module is available for the player.

Reply:
P.S it looks like we have everything we need for MVP at least and probably beyond, with regards to SMP so hopefully no more backend work to do other than cleanup tasks if theyre needed (and tests)

Reply:
I've changed this to export `BumpLoader` and import it in the `PortraitVideoCarousel`.

Basically does the same thing you had, but from a central location. Probably better ways to only load it in when the user opens a video, but fine for PoC.